src/snowplow/prospect/testData.ts

```diff
@@ -22,8 +49,8 @@ export const testProspectData = {
   title: 'A tale of two cities',
   isSyndicated: true,
   isCollection: false,
-  authors: 'Charles Dickens, Mark Twain',
-  prospectReviewStatus: ProspectReviewStatus.Dismissed,
```

Comment:
here we pretty much have same status sent by prospect-api so changing this to string, makes it easier to assign event-bridge.prospect to snowplow.prospect as other fields remain the same